static/app/views/organizationGroupDetails/groupDetails.spec.jsx

```diff
@@ -67,11 +68,20 @@ describe('groupDetails', () => {
 
   const createWrapper = (props = {selection}) => {
     return render(
-      <OrganizationContext.Provider value={organization}>
-        <GroupDetails {...router} selection={props.selection}>
-          <MockComponent />
-        </GroupDetails>
-      </OrganizationContext.Provider>,
+      <RouteContext.Provider
```

Comment:
Is there a reason why this is needed with `context: routerContext` provided below?

Reply:
Because we don't correctly pass that yet... we should fix